src/condition.ts

```diff
@@ -5,12 +5,14 @@ import { ConditionTarget } from "./condition-target";
 import { PropertyPath } from "./property-path";
 import { ObservableArray } from "./observable-array";
 import { Property } from "./property";
+import { Rule } from "./rule";
 
 export class Condition {
 	type: ConditionType;
 	message: string;
 	origin: string;
```

Comment:
Another pre-existing issue is that `origin` does not appear to be used

Reply:
fixed!